examples/credential_metadata_sd_jwt_vc.json

```diff
@@ -43,19 +45,17 @@
                         }
                     ]
                 },
-                "email": {},
-                "phone_number": {},
-                "address": {
-                    "street_address": {},
-                    "locality": {},
-                    "region": {},
-                    "country": {}
-                },
-                "birthdate": {},
-                "is_over_18": {},
-                "is_over_21": {},
-                "is_over_65": {}
-            }
+                {"path": ["email"]},
+                {"path": ["phone_number"]},
+                {"path": ["address", "street_address"]},
```

Comment:
We would like to give display properties to an object of claims. The wallet could use this property when collapsing the object's claims in the UI. Is this possible and would the following be the right way?
If so, this should be part of an example.

```json
                  {
                    "path": ["address"],
                    "display": [
                        {
                            "name": "Address",
                            "locale": "en-US"
                        },
                        {
                            "name": "Adresse",
                            "locale": "de-DE"
                        }
                    ],
                    "value_type": "object"
                },
                {"path": ["address", "street_address"]},
```

Reply:
+1 for this! I think supporting this is mandatory for having a clear and defined way to present this kind of data to the user. Let's use the example from your documentation and extend it with an arbitrary example:
```
... 
"address": {
    "street_address": "42 Market Street",
    "city": "Milliways",
    "postal_code": "12345"
},
...
"work_address":  {
   "city": "Berlin",
  ...
},
```

I would expect/imagine this to be displayed in the ui similar to this:

```
place of residence
+- street: 42 Market Street
+- city: Milliways
`- post code: 12345
....
place of work
+- city: Berlin
+...
```